pkgs/applications/science/misc/boinc/default.nix

```diff
@@ -26,7 +26,7 @@
 
 let
   majorVersion = "7.20";
-  minorVersion = "2";
+  minorVersion = "5";
```

Comment:
This could use lib.versions.major/minor/majorMinor

Reply:
Thanks for the suggestion!